addons/timecop/bootstrap.js

```diff
@@ -0,0 +1,40 @@
+const {classes: Cc, interfaces: Ci, utils: Cu} = Components;
+Cu.import("resource://gre/modules/Services.jsm");
+
+const BROWSER_SEARCH_PREF = "browser.search.";
+
+let gInstall = false;
+
+function install(aData, aReason) {
+  // We check for install here instead of in startup because
+  // In Firefox 46, the install reason didn't work properly
+  // in startup in some cases.
+  if (aReason != 5 /* ADDON_INSTALL */) {
+    return;
+  }
+  gInstall = true;
+}
+
+function uninstall(aData, aReason) {}
+
+function startup(aData, aReason) {
+  if (!gInstall) {
+    return;
+  }
+  Services.prefs.setBoolPref(BROWSER_SEARCH_PREF + "reset.geo", true);
```

Comment:
Why not use `getBranch()` here and below?
Not really important since this is just one-shot code, mostly just curious.

Reply:
Honestly it's just convenience (and copying search code). I'll look at getBranch() just so I get used to it.